pkg/storage/ingest/reader.go

```diff
@@ -894,45 +900,57 @@ func (r *concurrentFetchers) pollFetches(ctx context.Context) (result kgo.Fetche
 	}
 }
 
+var errUnknownPartitionLeader = fmt.Errorf("unknown partition leader")
```

Comment:
nit: should we place this sat the top next to the other errors?

Reply:
good spot, moved